packages/client/ui/react-ui/package.json

```diff
@@ -29,6 +29,9 @@
         "@crossmint/client-sdk-smart-wallet": "workspace:*",
         "@crossmint/client-sdk-window": "workspace:*",
         "@crossmint/common-sdk-base": "workspace:*",
+        "@dynamic-labs/ethereum": "3.2.0",
```

Comment:
important for this tree shaking pr to land soon, so these packages wont have any impact on other components in the sdk

it will be merged soon
https://github.com/Crossmint/crossmint-sdk/pull/788